### ISSUE TITLE
Ambient definitions in typeRoots are not automatically included

### ISSUE THREAD

**Issue by screendriver** [fathyb/parcel-plugin-typescript#19]
In my `tsconfig.json` I have custom typings defined:

```json
{
  "compilerOptions": {
    // ...
    "typeRoots": ["node_modules/@types", "typings"]
  }
}
```

so I can declare my own types in the `typings` directory for libraries that doesn't have one at the moment. If I ran `tsc` or `parcel` everything works. If I run it with this plugin I get a lot of `Could not find a declaration file for module ...`

**Comment by fathyb** [comment]
I tried and it worked, could you provide a reproduction repository?

**Comment by screendriver** [comment]
I tried to create a fresh repo for you but now I get a different error from your plugin:

```
TypeError: Cannot destructure property `diagnostics` of 'undefined' or 'null'.
    at TSAsset.<anonymous> (/Users/me/Development/parcel-sandbox/node_modules/parcel-plugin-typescript/build/frontend/assets/typescript.js:23:45)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/me/Development/parcel-sandbox/node_modules/tslib/tslib.js:104:62)
    at <anonymous>
```

Without `parcel-plugin-typescript` everything works. You can find a dummy repo [here](https://github.com/screendriver/parcel-sandbox).

**Comment by fathyb** [comment]
Thanks for the repro repo! I can't reproduce the second error with your repo (`Cannot destructure property diagnostics of 'undefined' or 'null'`). I think this is a bug of `0.5.1` bug was fixed with `0.5.2`, could you run `yarn list parcel-plugin-typescript` to be sure you are running `0.5.2`? If it is `0.5.2` then you can try deleting the `.cache` directory.
 
But I can now reproduce the type error, I still need to do some investigation because I think that this is the intended behaviour since TypeScript 2.0 and it works with `tsc` because it handles `includes` differently. The temporary workaround is to add the types to your include directory like this :
```js
{
  "compilerOptions": {
    // ...
   },
  "include": ["src/**/*", "types/**/*"]
}
```

I still take it as a bug as I want the plugin to behave like `tsc` so it'll be probably fixed in the future. Maybe related to #11.

**Comment by screendriver** [comment]
Thank you! 👍
